### DISCUSSION TITLE
Support for Jetbrains IDE's 2024.2 EAP

### DISCUSSION THREAD

**Discussion by dmeehan1968** [community/community#126240] [Copilot Conversations]
### Select Topic Area

Question

### Body

Jetbrains released their Early Access Program 2024.2 release a couple of weeks ago, and the Copilot plugin has yet to be updated.  This is preventing me from updating/evaluating the EAP release.

When will copilot plugin be updated for this?

**Comment by Scott-G** [comment]
No official feedback on this?

This EAP has been released for 2 weeks and still no compatible plugin for CoPliot. If this is going to be the "new norm" I will be cancelling my subscription.

**Comment by rhydianroberts** [comment]
The latest nightly now supports 2024.2: https://plugins.jetbrains.com/plugin/17718-github-copilot/versions/nightly/555078
